pcweb/templates/docpage/docpage.py

```diff
@@ -97,6 +97,9 @@ def docpage_footer(path: str):
     from pcweb.pages.docs.gallery import gallery
     from pcweb.pages.docs import getting_started, hosting
     from pcweb.pages.changelog import changelog
+
+    
+    path = path[:-1]
```

Comment:
We should do `path.strip("/")` just in case we pass one that doesn't have the trailing slash

Reply:
path.strip("/") is not working as the path is a baseVar in the state, hence I have done a work around to check if there is a trailing slash using rx.cond and then have removed the last one if there is one. The problem seems to have been resolved hence it will handle both the case when there is no trailing slash and when there is one